mssql-ballerina/tests/init-test-scripts.bal

```diff
@@ -0,0 +1,360 @@
+
+    DROP TABLE IF EXISTS ApproximateNumeric;
+
+    CREATE TABLE ApproximateNumeric(
+    row_id INT PRIMARY KEY,
+    float_type float,  
+    real_type real
+    );
+
+    INSERT INTO ApproximateNumeric (row_id, float_type, real_type) VALUES
+    (1, 1.79E+308, -1.18E-38);
+
+    DROP TABLE IF EXISTS DateandTime;
+
+    CREATE TABLE DateandTime(
+    row_id INT PRIMARY KEY,
+    Date_col  date, 
+    DateTimeOffset_col  datetimeoffset,
+    DateTime2_col datetime2, 
+    SmallDateTime_col smalldatetime, 
+    DateTime_col datetime,
+    Time_col time
+    );
+
+    INSERT INTO DateandTime (row_id, Date_col, DateTimeOffset_col, DateTime2_col, SmallDateTime_col , DateTime_col, Time_col)
+     VALUES (1, '2017-06-26', '2020-01-01 19:14:51', '1900-01-01 00:25:00.0021425', '2007-05-10 10:00:20', '2017-06-26 09:54:21.325', '09:46:22');
+
+    DROP TABLE IF EXISTS StringTypes;
+    
+    CREATE TABLE StringTypes (
+        row_id INT PRIMARY KEY,
+        varchar_type VARCHAR(255),
+        char_type CHAR(4),
+        text_type TEXT,
+        nchar_type NCHAR(4),
+        nvarchar_type NVARCHAR(10)
+    );
+
+    INSERT INTO StringTypes (row_id, varchar_type, char_type, text_type, nchar_type, nvarchar_type) VALUES (1,'str1','str2','assume a long text','str4','str5');
+
+    `;
+    _ = executeQuery("simple_params_query_db", query);
+}  
+
+
+public function createQuery(sql:ParameterizedQuery query) {
+
+    Client|sql:Error mssqlClient = new(user="sa",password="Test123#");
+
+    if(mssqlClient is sql:Error) {
+        io:println("Client init failed\n",mssqlClient);
+    }
+    else{
+        sql:ExecutionResult|sql:Error result__;
+        sql:Error? e__;
+
+        result__ = mssqlClient->execute(query);
+        if(result__ is sql:Error) {
+            io:println("Init Database drop failed\n",result__);
+        }
+        else{
+            io:println("Init Database drop passed\n",result__);
+        }
+        e__ = mssqlClient.close();
+
+        if(e__ is sql:Error) {
+            io:println("Client close failed\n",e__);
+        }
+        else{
+            io:println("Client closed");
+        }
+    }
+
+}
+
+public function executeQuery(string database, sql:ParameterizedQuery query) {
+
+    Client|sql:Error mssqlClient = new(user="sa",password="Test123#", database = database);
+
+    if(mssqlClient is sql:Error) {
+        io:println("Client init failed\n",mssqlClient);
+    }
+    else{
+        sql:ExecutionResult|sql:Error result__;
+        sql:Error? e__;
+
+        result__ = mssqlClient->execute(query);
+        if(result__ is sql:Error) {
+            io:println("Init Execute drop failed\n",result__);
+        }
+        else{
+            io:println("Init Execute drop passed\n",result__);
+        }
+        e__ = mssqlClient.close();
+
+        if(e__ is sql:Error) {
+            io:println("Client close1 failed\n",e__);
+        }
+        else{
+            io:println("Client closed");
+        }
+    }
```

Comment:
Can you explain a bit about this file?

Reply:
This file has bal functions to execute mssql scripts as I couldn't find a way to execute mssql scripts using DockerFile.